src/xenia/kernel/xam/apps/xmp_app.h

```diff
@@ -113,6 +109,12 @@ class XmpApp : public App {
   uint32_t next_song_handle_;
 };
 
+enum : XNotificationID {
+  kMsgStateChanged = 0x0A000001,
```

Comment:
This should be kept in `XmpApp` (as these are messages specific to `XmpApp`, not global messages for all apps — while in the commit it's just `apps::kMsgStateChanged` without clear association to a specific app), though maybe in the `public` part even since they may be used to send messages to `XmpApp` from other places, not just for receiving them.

Reply:
Okay. I've moved the enum to the `public` part of the `XmpApp` class,